src/unsafe-eval/init.ts

```diff
@@ -0,0 +1,25 @@
+import { GlUniformGroupSystem } from '../rendering/renderers/gl/shader/GlUniformGroupSystem';
+import { generateUniformsSyncPolyfill } from './generateUniformsSyncPolyfill';
+
+import type { IUniformData } from '../rendering/renderers/gl/shader/GlProgram';
+import type { UniformGroup } from '../rendering/renderers/shared/shader/UniformGroup';
+import type { UniformsSyncCallback } from '../rendering/renderers/shared/shader/utils/createUniformBufferSync';
+
+export function selfInstall()
+{
+    Object.assign(GlUniformGroupSystem.prototype,
+        {
+            _systemCheck()
+            {
+                // Do nothing, don't throw error
+            },
+
+            _generateUniformsSync(group: UniformGroup, uniformData: Record<string, IUniformData>): UniformsSyncCallback
+            {
+                return generateUniformsSyncPolyfill(group, uniformData);
```

Comment:
you don't need to generate this each time as the the single polyfill function should cover every shader. You can basically return the same function every time here!